src/nih_wayfinding/test/spec/geocoder/search-service.spec.js

```diff
@@ -9,10 +9,29 @@ describe('nih.geocoder: Geocoder', function () {
 
     var httpBackend;
     var Geocoder;
-    var searchUrl = 'http://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer/find';
+    var searchUrl = 'http://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer/find?bbox=-87.940101,41.643919,-87.523984,42.023022&category=Address,Postal&f=pjson&maxLocations=1&outFields=StAddr,City,Postal&text=one';
     var searchResponse = {
```

Comment:
Since we'll be using this to actually provide directions, wouldn't we want to return more than once search result, in case the first response isn't what they're looking for?  We probably shouldn't rely on everyone pausing to look through suggestions to select.